docker-compose.yml

```
@@ -522,16 +573,19 @@ services:
       - serverless-lambda-no-param-sync
       - serverless-lambda-one-param-sync
       - serverless-lambda-two-params-sync
+      - serverless-lambda-no-param-sync-with-context
+      - serverless-lambda-one-param-sync-with-context
+      - serverless-lambda-two-params-sync-with-context
       - serverless-lambda-no-param-async
       - serverless-lambda-one-param-async
       - serverless-lambda-two-params-async
       - serverless-lambda-no-param-void
       - serverless-lambda-one-param-void
       - serverless-lambda-two-params-void
-      - serverless-integration-trace-context-mock
+      - serverless-integration-extension-mock
```

I think you need to add the `with-context` dependency to get fix the snapshot issues

```suggestion
      - serverless-integration-extension-mock
      - serverless-integration-extension-mock-with-context
```